docs/ota-client-guide/modules/ROOT/pages/garage-sign.adoc

```diff
@@ -0,0 +1,256 @@
+
+* *Offline keys*
++
+In a production environment, we recommend that you keep the keys for signing software versions offline. Therefore, the OTA Connect server will no longer sign software for you, instead, they will be signed locally by your machine. This maintains the security of the software because if someone gained access to your OTA Connect account, they will be able to send malicious software to your devices. However, if your keys are offline, your devices will not recognize the software as it will not have the correct credentials. This is where `garage-sign` becomes useful. For non-OSTree images, the `garage-sign` tool is used to manage and rotate your keys offline as well as sign your software.
+
+
+* *Local repository*
++
+A repository is a location for storing software packages. A local repository is a server that exists on your local machine such as your computer. When using the `garage-sign` tool, it is important to create a local repository on your machine that contains your credentials from OTA Connect, and your keys for signing metadata. With your keys existing locally on your machine, any changes you make to them will not directly affect your devices on OTA Connect. It also ensures the security of your keys during production.
+
+****
+
+
+== Prerequisites of `garage-sign`
+
+. Download the latest version of the `garage-sign` tool from link:https://tuf-cli-releases.ota.here.com/index.html[here].
+. Unzip the file you have downloaded to a location on your computer.
+. Change the directory of your terminal on the command line to that location.
+. Afterwards, open the `garage-sign` folder in your terminal. You can also print the help command to view the list of all possible commands with this tool.
++
+[source,bash]
+----
+cd garage-sign/
+bin/garage-sign --help
+----
+
+For more information, please refer to the link:https://github.com/advancedtelematic/ota-tuf/tree/master/cli[`garage-sign` README file].
+
+
+== Use of `garage-sign` with OTA Connect
+
+To use the `garage-sign` tool with OTA Connect, you need to perform the following steps:
+
+. Initialize a new image repository locally on your computer that pulls your `credentials.zip` from OTA Connect.
++
+[source, bash]
+----
+garage-sign init \
+  --repo <reponame> \ <1>
+  --credentials </path/to/credentials.zip> <2>
+----
++
+<1> The name of your image repository which is to be initialized by the `init` command is provided here.
+<2> Here you provide the path of your `credentials.zip` file downloaded from your account on the https://connect.ota.here.com[OTA Connect Portal].
+
+. Ensure you have the latest `targets.json` file by pulling it from OTA Connect.
++
+[source, bash]
+----
+garage-sign targets pull --repo <reponame>
+----
+
+The above commands create a local repository in a directory called `/tuf/<reponame>`. This directory will contain your `credentials.zip` file and has a tree structure as shown below:

```

Comment:
That leading slash might cause some confusion.
```suggestion
The above commands create a local repository in a directory called `tuf/<reponame>`. This directory will contain your `credentials.zip` file and has a tree structure as shown below:

```

Reply:
@patrickvacek good catch. Suggestion applied